test/Microsoft.IdentityModel.Tokens.Tests/Json/DataSets.cs

```diff
@@ -732,6 +732,7 @@ public static JsonWebKeySet JsonWebKeySet1
                 JsonWebKeySet jsonWebKeySet = new JsonWebKeySet();
                 jsonWebKeySet.Keys.Add(JsonWebKey1);
                 jsonWebKeySet.Keys.Add(JsonWebKey2);
+                jsonWebKeySet.JsonData = JsonWebKeySetString1;
```

Comment:
Can we use the JsonWebKeySet constructor that accepts JSON instead of setting the JsonData and adding keys here? That way we'll know JSON data and keys are equivalent; plus an extra check that `JsonWebKeySetSerializer` reads the string properly.